tg/models.py

```diff
@@ -516,3 +550,48 @@ def get_user(self, user_id: int) -> Dict[str, Any]:
             return {}
         self.users[user_id] = result.update
         return result.update
+
+    def get_group_info(self, group_id):
+        if group_id in self.groups:
+            return self.groups[group_id]
+        self.tg.get_basic_group(group_id)
+
+    def get_supergroup_info(self, supergroup_id):
+        if supergroup_id in self.supergroups:
+            return self.supergroups[supergroup_id]
+        self.tg.get_supergroup(supergroup_id)
+
+
+def pretty_ts(ts):
+    from datetime import datetime
```

Comment:
why local import? also, this function should  be moved to some better place, maybe in the utils module